benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1721,7 +1727,7 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
     switch (pFullscreenQuadsType->GetValue()) {
         case FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_NOISE: {
             uint32_t noiseQuadRandomSeed = (uint32_t)seed;
-            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[0], 1, &noiseQuadRandomSeed);
+            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[0], sizeof(uint32_t) / sizeof(uint32_t), &noiseQuadRandomSeed, sizeof(uint32_t) / 4);
```

Comment:
would it be cleaner to have a helper function, something like `size_t GetPushConstCount<T>()` so that we could use function everywhere we need to get the count?